hoverxref/_static/js/hoverxref.js_t

```diff
@@ -44,18 +44,37 @@ function reLoadSphinxTabs() {
             // HACK: please, improve this code to call the content of "tab.js" without creating a script element
 
             // Get the URL from the current generated page since it's not always the same
-            var src = $('script[src$="sphinx_tabs/tabs.js"]')[0].src;
+            var src = $('script[src$="sphinx_tabs/tabs.js"]');
+            if (src.length != 0) {
+                // sphinx-tabs < 2
+                src = src[0].src
+                script = d.createElement('script');
+                script.type = 'text/javascript';
+                script.onload = function(){
+                    // remote script has loaded
+                };
+                script.src = src;
+                d.getElementsByTagName('head')[0].appendChild(script);
 
-            script = d.createElement('script');
-            script.type = 'text/javascript';
-            script.onload = function(){
-                // remote script has loaded
-            };
-            script.src = src;
-            d.getElementsByTagName('head')[0].appendChild(script);
+                // Once the script has been executed, we remove it from the DOM
+                script.parentNode.removeChild(script);
+            }
+            var src = $('script[src$="_static/tabs.js"]');
```

Comment:
Should probably use a different variable for this?

Reply:
I renamed them to `older_tab_src` and `newer_tab_src`.